arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -1485,6 +1888,13 @@
       "required": true,
       "type": "string",
       "description": "Client API version."
+    },
+    "PredefinedPolicyNameParameter": {
+      "name": "predefined-poicy-name",
```

Comment:
This does not match the name defined in the path

`/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableSslOptions/default/predefinedPolicies/{predefinedPolicyName}`

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/241395216#L1165